sources/packages/api/src/route-controllers/supporting-user/supporting-user.aest.controller.ts

```diff
@@ -33,4 +40,49 @@ export class AESTSupportingUserController {
         } as ApplicationSupportingUsersDTO),
     );
   }
+
+  /**
+   * Get supporting user formName and the form data
+   * with the supportingUserId
+   * @param supportingUserId supporting user id.
+   * @param applicationId application id.
+   * @returns supporting user form data and details.
+   */
+  @Get(":supportingUserId/application/:applicationId")
```

Comment:
Do we need the application id to be passed in the URL when the supporting user id can give us this information?
The method getSupportingUsersDetails can be called before the _getSupportingUserFormName_.

Reply:
Thanks for doing this change 😉